src/usr/local/www/system_usermanager_settings.php

```diff
@@ -102,6 +102,8 @@
 
 $pconfig['backend'] = &$config['system']['webgui']['backend'];
 
+$pconfig['auth_refresh_time'] = &$config['system']['webgui']['auth_refresh_time'];
```

Comment:
Why set it to a reference of $config? I believe it's not necessary the way code is implemented